app/Resources/assets/js/jquery.instantSearch.js

```diff
@@ -0,0 +1,79 @@
+// jQuery plugin for an instant searching
```

Comment:
is this a plugin you wrote specifically for the Symfony demo, or some third-part plugin ?